GFS_layer/GFS_physics_driver.F90

```diff
@@ -718,11 +718,8 @@ subroutine GFS_physics_driver_down                         &
       smsoil(:,:) = Sfcprop%smc(:,:)
       stsoil(:,:) = Sfcprop%stc(:,:)
       slsoil(:,:) = Sfcprop%slc(:,:)          !! clu: slc -> slsoil
-      Statemid%stored_dudt(:,:)  = 0.
-      Statemid%stored_dvdt(:,:)  = 0.
-      Statemid%stored_dtdt(:,:)  = 0.
-      Statemid%stored_dtdtc(:,:) = 0.
-      Statemid%stored_dqdt(:,:,:) = 0.
+
```

Comment:
I didn't know Fortran supported member subroutines in structures like in C++'s classes. This is convenient.